packages/frontend/components/Providence/Feed.vue

```diff
@@ -36,51 +36,42 @@ export default {
             type: String,
             default: "",
         },
+        provenance: {
+            default: null,
+        },
     },
     data() {
         return {
-            reports: [],
             attachmentURLs: {},
         };
     },
-    created() {
+    mounted() {
         EventBus.on('feedRefresh', this.refreshPage);
         this.refreshPage();
     },
-    beforeDestroy() { 
+    beforeDestroy() {
         EventBus.off('feedRefresh', this.refreshPage);
     },
     methods: {
         async fetchAttachmentsForReport(report, index) {
             try {
                 if (report.attachments.length > 0) {
-                    const attachmentPromises = report.attachments.map(attachmentID => getAttachment(this.deviceKey, attachmentID));
+                    const baseUrl = useRuntimeConfig().public.baseUrl;
```

Comment:
Because this is now pulled from the getAttachment function, we should have more documentation in azureFuncs.ts file oh what you need to input into the function.

Reply:
What the baseUrl is not as self-explanatory as deviceKey etc.